package.json

```diff
@@ -9,16 +9,16 @@
     "test:browser:min": "MIN=true browserify --bare ./test | tape-run --node",
     "test:perf": "npm run minify && browserify --bare ./test/perf.js | tape-run",
     "build:demo": "browserify --bare ./demo > ./docs/bundle.js",
-    "minify": "uglifyjs index.js -c unused,dead_code,hoist_vars,loops=false,ecma=7,hoist_props=true,hoist_funs,toplevel,keep_classnames,keep_fargs=false -o dist/tonic.min.js"
+    "minify": "terser index.js -c unused,dead_code,hoist_vars,loops=false,hoist_props=true,hoist_funs,toplevel,keep_classnames,keep_fargs=false -o dist/tonic.min.js"
```

Comment:
TIL: `terser`

Reply:
hah! me too :) I guess `uglify-js` never supported `es6`, `uglify-es` does but is abandoned and `terser` is maintained and supports latest `es6`.